prov/util/src/util_shm.c

```diff
@@ -62,6 +62,26 @@ static void smr_peer_addr_init(struct smr_addr *peer)
 	peer->addr = FI_ADDR_UNSPEC;
 }
 
+void smr_cma_check(struct smr_region *smr, struct smr_region *peer_smr)
+{
+	struct iovec local_iov, remote_iov;
+	int ret;
+
+	if (peer_smr->cma_cap != -1) {
```

Comment:
It can be. As we need to execute a process_vm_writev call to try to write on the peer's cma_cap address, in the latest revision I set cma_cap as a separate flag which is different from the current `uint16_t flags`, so that we don't mess up the value in current flags. Is it okay?